src/views/Home/components/DaysLeft/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import Card, { CardContent } from 'material-ui/Card';
+import { Doughnut } from 'react-chartjs-2';
+
+import { gordonColors } from '../../../../theme';
+import session from '../../../../services/session';
+import GordonLoader from '../../../../components/Loader';
+
+
+export default class DaysLeft extends Component {
+  constructor(props) {
+    super(props);
+
+    this.getDaysLeft = this.getDaysLeft.bind(this);
+
+    this.state = {
+      daysLeft: [],
+      loading: true,
+    };
+  }
+  componentWillMount() {
+    this.getDaysLeft();
+  }
+  async getDaysLeft() {
+    const daysLeft = await session.getDaysLeft();
+    this.setState({ daysLeft });
+    this.setState({ loading: false });
+  }
+  render() {
+    let content;
+    const daysleft = this.state.daysLeft[0];
+    const pastDays = this.state.daysLeft[1] - daysleft;
+    const Data = {
+      datasets: [{ data: [pastDays, daysleft], backgroundColor: [gordonColors.primary.blue] }],
+    };
+    if (this.state.loading === true) {
```

Comment:
With this approach (in this component and the other one), the homepage just looks like two random loading spinners until the data has arrived. A better approach would be to swap out the `<Doughnut ... />` element for a loader, then replace the loader with the chart once the data has loaded. This way, there would be context for what the user is waiting for ("oh, my chapel credits are loading"), and the card would be around the loader, so it would look better.